pkg/connector/backfill.go

```diff
@@ -121,6 +124,10 @@ func (l *LinkedInClient) FetchMessages(ctx context.Context, fetchParams bridgev2
 		}
 
 		resp.Messages = append(resp.Messages, &backfillMessage)
+
+		if resp.MarkRead && msg.DeliveredAt.UnixMilli() > lastRead.UnixMilli() {
```

Comment:
If you have two `time.Time`s, you should just use the built-in comparison functions (`.After()` / `.Before()`) rather than getting unix time from both